lua/fff/picker_ui.lua

```diff
@@ -989,6 +989,8 @@ function M.close()
     if buf and vim.api.nvim_buf_is_valid(buf) then vim.api.nvim_buf_delete(buf, { force = true }) end
   end
 
+  main.stop_background_monitor()
```

Comment:
This is not an option to do because we have to keep the sync running in the background, we should clean it up when the neovim is requested to exit but I've been sure that it should already be cleaned up let me research this a little bit

Reply:
they spawning once per neovim session (at least they should be) I will try to wrap my head around this more deeply can you share the platform info